dev-docs/modules/userId.md

```diff
@@ -66,6 +66,7 @@ of sub-objects. The table below has the options that are common across ID system
 | storage.type | Required | String | Must be either `"cookie"` or `"html5"`. This is where the results of the user ID will be stored. | `"cookie"` |
 | storage.name | Required | String | The name of the cookie or html5 local storage where the user ID will be stored. | `"_unifiedId"` |
 | storage.expires | Optional | Integer | How long (in days) the user ID information will be stored. Default is 30 for UnifiedId and 1825 for PubCommonID | `365` |
+| storage.refreshInSeconds | Optional | Integer | The amount of time (in seconds) the user ID should be cached in storage before calling the provider again to retrieve a potentially updated value for their user ID. If set, it should be less than the time defined in `storage.expires`. By default the ID will not be refreshed until it expires.
```

Comment:
For clarity, I suggest changing the second sentence in the description to: "If set, this value should equate to a time period less than the number of days defined in `storage.expires`.

Reply:
updated